extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts

```diff
@@ -0,0 +1,642 @@
+			this.editVCores(loc.vCoresMin);
+			this.editVCores(loc.vCoresMax);
+			this.editMemory(loc.memoryMin);
+			this.editMemory(loc.memoryMax);
+
+			return [
+				this.workerNodesSectionContainer(),
+				this.vCoreMemorySection(),
+				this.configurationSectionContainer(loc.vCoresMin, this.vCoresMinBox!, '40px'),
+				this.configurationSectionContainer(loc.vCoresMax, this.vCoresMaxBox!, '40px'),
+				this.configurationSectionContainer(loc.memoryMin, this.memoryMinBox!, '40px'),
+				this.configurationSectionContainer(loc.memoryMax, this.memoryMaxBox!, '20px')
+
+			];
+
+		}
+
+
+	}
+
+	private workerNodesSectionContainer(): azdata.FlexContainer {
+		const inputFlex = { flex: '0 1 150px' };
+		const keyFlex = { flex: `0 1 250px` };
+
+		const flexContainer = this.modelView.modelBuilder.flexContainer().withLayout({
+			flexWrap: 'wrap',
+			alignItems: 'center'
+		}).component();
+
+		const keyComponent = this.modelView.modelBuilder.text().withProperties<azdata.TextComponentProperties>({
+			value: loc.workerNodeCount,
+			CSSStyles: { ...cssStyles.text, 'margin-block-start': '0px', 'margin-block-end': '0px' }
+		}).component();
+
+		const keyContainer = this.modelView.modelBuilder.flexContainer().withLayout({ alignItems: 'center' }).component();
+		keyContainer.addItem(keyComponent, { CSSStyles: { 'margin-right': '0px', 'margin-bottom': '15px' } });
+
+		const information = this.modelView.modelBuilder.button().withProperties<azdata.ButtonProperties>({
+			iconPath: IconPathHelper.information,
+			title: loc.workerNodesInformation,
+			width: '12px',
+			height: '12px',
+			enabled: false
+		}).component();
+
+		keyContainer.addItem(information, { CSSStyles: { 'margin-left': '5px', 'margin-bottom': '15px' } });
+		flexContainer.addItem(keyContainer, keyFlex);
+
+		const inputContainer = this.modelView.modelBuilder.flexContainer().withLayout({ alignItems: 'center' }).component();
+		inputContainer.addItem(this.workerBox!, { CSSStyles: { 'margin-bottom': '15px', 'min-width': '50px', 'max-width': '225px' } });
+
+		flexContainer!.addItem(inputContainer, inputFlex);
+
+		return flexContainer;
+	}
+
+	private configurationSectionContainer(key: string, input: azdata.Component, nestingLineHeight: string): azdata.FlexContainer {
+		const inputFlex = { flex: '0 1 150px' };
+		const keyFlex = { flex: `0 1 200px` };
+		const bottomLineFlex = { flex: `0 1 45px` };
+
+		const flexContainer = this.modelView.modelBuilder.flexContainer().withLayout({
+			flexWrap: 'nowrap',
+			alignItems: 'center'
+		}).component();
+
+		const leftLine = this.modelView.modelBuilder.divContainer().withProperties({
+			CSSStyles: { 'max-height': nestingLineHeight, 'min-height': nestingLineHeight, 'max-width': '1px', 'border-left-style': 'solid', 'border-left-color': '#ccc' }
+		}).component();
+
+		flexContainer!.addItem(leftLine, { CSSStyles: { 'align-self': 'flex-start' } });
+
+		const bottomLine = this.modelView.modelBuilder.divContainer().withProperties({
```

Comment:
Does this add 1 div for the left border and a 2nd div for the bottom border?  Is it possible to have just 1 div that sets both the left and bottom borders?

Reply:
When trying that out the bottom border would not align with the key titles or the left border would not be long enough to connect.